packages/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -58,7 +58,8 @@ MeetingUtil.remoteUpdateAudioVideo = (audioMuted, videoMuted, meeting) => {
     selfId: meeting.selfId,
```

Comment:
not strictly part of this pull request, but remoteAudioVideoToggle is now not well named, and is also lacking preferTrancoding from the docstring

Reply:
To be honest, I think this method has never been correctly named. My understanding is that this method informs Locus about the state of **local** mute of audio and video (it's called from `MeetingUtil.remoteUpdateAudioVideo()` which is called from `sendLocalMuteRequestToServer()`), so yeah it should be renamed, but such a fix should be done on master and not here as we want to minimize the risk of any merge conflicts for the sdk_v3 branch, so I won't do this in this PR